### PR TITLE
New module: CSS cascade module

### LINKED ISSUE #165: New module: CSS Cascading and Inheritance
- [ ] Create new page
- [ ] brief (5-paragraph approx) description
- [ ] demonstration example
- [ ]  properties
- [ ]  data types
- [ ]  associated properties
- [ ]  glossary terms
- [ ] Link to this page from appropriate pages

### PR DESCRIPTION
**Pull request by estelle**
Module page did not exist. So, created it.

Fixes https://github.com/openwebdocs/project/issues/165

(Note: this is part of a much larger project. The pseudo elements module link looks broken but the page is almost ready for PR so will be fixed when that PR (not yet committed) is merged.

### CODE REVIEW

**Comment by dipikabh** [comment]
Sorry for the delay on this, @estelle. Taking a look now.

File: files/en-us/web/css/css_cascade/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+slug: Web/CSS/CSS_cascade
+page-type: css-module
+spec-urls:
+  - https://drafts.csswg.org/css-cascade/
+  - https://drafts.csswg.org/css-cascade-5/
+---
+
+{{CSSRef}}
+
+The **CSS cascade and inheritance** module defines the rules for assigning property values, cascading, and inheritance. This module specifies the rules for finding the specified value for all properties on all elements.
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:12]: how about slight rephrasing to show the relationship

```suggestion
The **CSS cascade and inheritance** module defines the rules for assigning values to properties by way of cascading and inheritance. This module specifies the rules for finding the specified value for all properties on all elements.
```

File: files/en-us/web/css/css_cascade/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+One of the fundamental design principles of CSS is cascading, which allows several style sheets to influence the presentation of a document. CSS property-value declarations define how a document is rendered. Multiple declarations may set different values for the same element/property combination, but only one value can be applied for any CSS property. The CSS cascade module defines how these conflicts are resolved.
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:14]:
```suggestion
One of the fundamental design principles of CSS is cascading of rules. It allows several style sheets to influence the presentation of a document. CSS property-value declarations define how a document is rendered. Multiple declarations may set different values for the same element and property combination, but only one value can be applied to any CSS property. The CSS cascade module defines how these conflicts are resolved.
```

File: files/en-us/web/css/css_cascade/index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+> **Note:** The rules for finding the specified values in the page context and its margin boxes are described in the [CSS page module](/en-US/docs/Web/CSS/CSS_paged_media).
+
+## Reference
+
+### Properties
+
+- {{cssxref("all")}}
+
+### at-rules
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:26]:
```suggestion
### At-rules
```

File: files/en-us/web/css/css_cascade/index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+- {{cssxref("inherit")}}
+- {{cssxref("revert")}}
+- {{cssxref("revert-layer")}} {{Experimental_Inline}}
+- {{cssxref("unset")}}
+
+### Concepts
+
+- {{cssxref("Specificity")}}
+- {{cssxref("important", "!important")}}
+- [Cascade origin and importance](/en-US/docs/Web/CSS/Cascade)
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:43]: These need to be covered under "Related concepts" or "Guides" section below.
- "Cascade origin and importance" is getting covered under "Guides" by "Introducing the CSS Cascade"
- "Specificity" can be moved to "Guides"

```suggestion

```

File: files/en-us/web/css/css_cascade/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+- [Values](/en-US/docs/Web/CSS/Value_definition_syntax)
+  - [actual value](/en-US/docs/Web/CSS/actual_value)
+  - [computed value](/en-US/docs/Web/CSS/computed_value)
+  - [initial value](/en-US/docs/Web/CSS/initial_value)
+  - [resolved value](/en-US/docs/Web/CSS/resolved_value)
+  - [specified value](/en-US/docs/Web/CSS/specified_value)
+  - [used value](/en-US/docs/Web/CSS/used_value)
+- [Origin types](/en-US/docs/Web/CSS/Cascade#origin_types)
+  - [user-agent origin](/en-US/docs/Web/CSS/Cascade#user-agent_stylesheets)
+  - [quthor origin](/en-US/docs/Web/CSS/Cascade#author_stylesheets)
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:63]:
```suggestion
  - [author origin](/en-US/docs/Web/CSS/Cascade#author_stylesheets)
```

File: files/en-us/web/css/css_cascade/index.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+## Specifications
+
+{{Specifications}}
+
+## See also
+
+- [CSS selectors module](/en-US/docs/Web/CSS/CSS_selectors)
+- [CSS pseudo-elements module](/en-US/docs/Web/CSS/CSS_pseudo)
+- [CSS page module](/en-US/docs/Web/CSS/CSS_paged_media)
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:101]:
```suggestion
- [CSS paged media module](/en-US/docs/Web/CSS/CSS_paged_media)
```

File: files/en-us/web/css/css_cascade/index.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+- [Cascade origin and importance](/en-US/docs/Web/CSS/Cascade)
+
+### Interfaces
+
+- {{DOMXRef("CSSLayerBlockRule")}}
+- {{DOMXRef("CSSGroupingRule")}}
+- {{DOMXRef("CSSLayerStatementRule")}}
+- {{DOMXRef("CSSRule")}}
+
+### Definitions
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:52]: Can we remove this section and move all its content to the "Related concepts" section - for consistency with other module landing pages

File: files/en-us/web/css/css_cascade/index.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+- [Declaring layers](/en-US/docs/Web/CSS/@import#importing_css_rules_into_a_cascade_layer)
+  - [named layer](/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers#the_layer_statement_at-rule_for_named_layers)
+  - [anonymous layer](/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers#the-layer-block-at-rule-for-named-and-anonymous-layers)
+- Glossary: {{glossary("style origin")}}
+
+## Guides
+
+- [Introducing the CSS Cascade](/en-US/docs/Web/CSS/Cascade)
+
+  - : Guide to the cascade algorithm that defines how user agents combine property values originating from different sources
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:74]:
```suggestion
  - : Guide to the cascade algorithm that defines how user agents combine property values originating from different sources.
```

File: files/en-us/web/css/css_cascade/index.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+- [Learn: Cascade, specificity, and inheritance](/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance)
+
+  - : The most fundamental concepts of CSS — the cascade, specificity, and inheritance — which control how CSS is applied to HTML and how conflicts are resolved.
+
+- [Learn: Cascade layers](/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers)
+
+  - : Introduction to [cascade layers](/en-US/docs/Web/CSS/@layer), a more advanced feature that builds on the fundamental concepts of the [CSS cascade](/en-US/docs/Web/CSS/Cascade) and [CSS specificity](/en-US/docs/Web/CSS/Specificity).
+
+- [CSS inheritance](/en-US/docs/Web/CSS/Inheritance)
+  - : A guide to CSS inheritance
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:85]:
```suggestion
  - : A guide to CSS inheritance.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/css_cascade/index.md (hunk 4 above)
Comment (estelle) [reply files/en-us/web/css/css_cascade/index.md]: moved important to keywords and the two others to definitions.

File: files/en-us/web/css/css_cascade/index.md (hunk 7 above)
Comment (estelle) [reply files/en-us/web/css/css_cascade/index.md:47]: these aren't "related". They're in this spec.

File: files/en-us/web/css/css_cascade/index.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+### Properties
+
+- {{cssxref("all")}}
+
+### At-rules
+
+- {{cssxref("@import")}}
+- {{cssxref("@layer")}}
+
+### Keywords
```
Comment (dipikabh) [inline files/en-us/web/css/css_cascade/index.md:31]: Should we call this section "Keyword values" to be more specific

File: files/en-us/web/css/css_cascade/index.md (hunk 7 above)
Comment (dipikabh) [reply files/en-us/web/css/css_cascade/index.md:47]: How about calling this section "Key concepts"?

**Review by dipikabh — APPROVED** [review]
Leaving two questions for your consideration and also a +1 as it is in good shape overall. Thank you!

File: files/en-us/web/css/css_cascade/index.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+- {{cssxref("important", "!important")}} flag
+
+### Interfaces
+
+- {{DOMXRef("CSSLayerBlockRule")}}
+- {{DOMXRef("CSSGroupingRule")}}
+- {{DOMXRef("CSSLayerStatementRule")}}
+- {{DOMXRef("CSSRule")}}
+
+### Definitions
```
Comment (estelle) [inline files/en-us/web/css/css_cascade/index.md:47]:
```suggestion
### Key concepts and definitions
```

**Pull request merged**